OptimoveSDK/optimove-sdk/src/main/java/com/optimove/android/optimobile/SessionHelper.java

```diff
@@ -58,6 +61,33 @@ public void activityAvailable(@NonNull Activity activity) {
         }
     }
 
+    private void checkNotificationEnablementForStatusChange(){
+        if (Build.VERSION.SDK_INT < Build.VERSION_CODES.TIRAMISU) {
```

Comment:
I think we maybe want to check on all possible API levels? This would mean we can track users who disable notifications in settings on earlier API levels too.